config/config.go

```diff
@@ -103,7 +101,6 @@ type Config struct {
 func SetDefaultConfigValues(v *viper.Viper) {
 	v.SetDefault(LogLevelKey, logging.Info.String())
 	v.SetDefault(StorageLocationKey, "./.awm-relayer-storage")
-	v.SetDefault(ProcessMissedBlocksKey, true)
```

Comment:
Please revert this.

Reply:
Sorry, must have happened with the merge. Added back with the default value changed to false

Reply:
We're actually going to keep these key to default to true. I'm about to open a PR to fix your issue a slightly different way.